app/views/directives/notifications/notification-footer.html

```diff
@@ -0,0 +1,8 @@
+<a
+  ng-if="$ctrl.customScope.hasUnread(notificationGroup)"
+  class="btn btn-link btn-block"
+  role="button"
+  ng-click="$ctrl.customScope.clearAll(notificationGroup)">
+  <span class="pficon pficon-close"></span>
+  <span> Clear All</span>
+</a>
```

Comment:
Using the show-clear-all would allow you to remove the footer if the only think in the footer is the clear all button.